content/departments/security/admin-access-internal-instances.md

```diff
@@ -7,6 +7,17 @@ Site-admin access to internal instances (dotcom, s2, rctest, demo, k8s) is provi
 
 ## How it works
 
+### New direct method
+
+Site-admin access can now be granted instantly using our new Entitler service. You can request the bundle by typing `/access_request` and in the 'Search permission' box type 'dotcom'. You will see the set 'Dotcom direct site-admin'. Set your desired duration, make sure you add a nice justification and you're good to go! Access should be granted within less than a minute.
+
+> [!NOTE]
+> This requires users to have a _verified_ @Sourcegraph.com email account as their primary email.
+
+If you have any issues with the integration, please drop a message in #discuss-security.
+
+### Deprecated SOAP method
```

Comment:
I'd say we can remove this entirely to avoid confusion when to use which.

Reply:
Ah I pushed that but

```
To github.com:sourcegraph/handbook.git
 ! [rejected]            vincent/update-site-admin-docs -> vincent/update-site-admin-docs (fetch first)
error: failed to push some refs to 'github.com:sourcegraph/handbook.git'
hint: Updates were rejected because the remote contains work that you do not
hint: have locally. This is usually caused by another repository pushing to
hint: the same ref. If you want to integrate the remote changes, use
hint: 'git pull' before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```

Thanks!